includes/fbbot/class-omise-fbbot-request-handler.php

```diff
@@ -0,0 +1,157 @@
+<?php
+defined( 'ABSPATH' ) or die( "No direct script access allowed." );
+
+if ( class_exists( 'Omise_FBBot_Request_Handler' ) ) {
+	return;
+}
+
+class Omise_FBBot_Request_Handler {
+	
+	private function __construct() {
+		// Hide the constructor
+	}
+
+	public static function handle_message_from( $sender_id, $user_message ) {
+		$message_store = Omise_FBBot_Message_Store;
+		$message = strtolower( $user_message );
+
+		if ( $message_store::check_greeting_words( $message ) ) {
```

Comment:
@igroomgrim just minor but if `Omise_FBBot_Conversation_Generator` is a class that contains logic to generate a proper sentence and `Omise_FBBot_Message_Store` is just a storage of sentences then I think this `checking` thing should be in `Omise_FBBot_Conversation_Generator`.

What do you think?